ecommerce/taxes/test/taxes_test.rb

```diff
@@ -3,51 +3,74 @@
 module Taxes
   class TaxesTest < Test
     def test_setting_available_vat_rate
+      vat_rate = Infra::Types::VatRate.new(code: "50", rate: 50)
+      add_available_vat_rate(vat_rate)
+
       product_id = SecureRandom.uuid
-      vat_rate_set = VatRateSet.new(data: { product_id: product_id, vat_rate: available_vat_rate })
+      vat_rate_set = VatRateSet.new(data: { product_id: product_id, vat_rate: vat_rate })
```

Comment:
small tip, you could do 
```
vat_rate_set = VatRateSet.new(data: { product_id:, vat_rate: })
```

instead :)

Reply:
I would prefer to leave this as is for now, as it is consistent with the rest of the code in the project. In the future we can add a rubocop rule to enforce style guides :)